### ISSUE TITLE
Server admin report for store's balance

### ISSUE THREAD

**Issue by pavlenex** [btcpayserver/btcpayserver#5675]
This issue shouldn't be tackled until https://github.com/btcpayserver/btcpayserver/issues/5674 is added. Also it is planned for 1.14

This issue is building on feedback from https://github.com/btcpayserver/btcpayserver/issues/396

**Problem**: Some server admins provide services to users using their instances. Some even have subscription model where they charge for support based on store's revenue. However, currently ,it's not possible for server admins to have easy insight in balances/revenue of the stores on their server.

**Solution**: Create a dedicated report in `Reporting` that's only available to server admin, that could showcase balances of stores on their server for a specific time-period.

**Blockers**: #5674 and 1.13.0 release.

**Scope**: We would have to discuss the scope of this issue on a dev call first and learn from ambasadors what kind of reporting would they need.

**Comment by dstrukt** [comment]
Makes sense, would we still want to show a snapshot of the current balance on the server user list as well? Or would we solely display that (current & historical) information on the report?

**Comment by pavlenex** [comment]
@dstrukt No, not for now. We agreed reports are the way for this at the moment and will follow along the feedback.

**Comment by pavlenex** [comment]
We plan to improve and in a way redefine reporting, closing this issue since the tracker is here, if anyone has specific feedback on what they need from reporting, feel free to comment there. https://github.com/btcpayserver/btcpayserver/issues/6716
